### DISCUSSION TITLE
can we set Icons, Emojis size?

### DISCUSSION THREAD

**Discussion by 91it** [squidfunk/mkdocs-material#7594] [Q&A] [answered]
We know font-awesome can set the icons size, but how to set the icons size in mkdocs-material? because mkdocs-material's icons include font-awesome
![image](https://github.com/user-attachments/assets/71073109-1bc0-4ba7-b84e-99de3a7b668f)

**Comment by squidfunk** [comment, marked as answer]
Yes, there are some [classes to influence the size](https://github.com/squidfunk/mkdocs-material/blob/25b2107b53877c32b55b1ea7620bd1c4565176fc/src/templates/assets/stylesheets/main/extensions/pymdownx/_emoji.scss#L56-L74), e.e. `.lg`, `.xl` or `.xxl`, but there still not stabilized, which is why they are currently undocumented. We'll likely keep them, but this still needs some work, mainly in terms of alignment. Use it like this:

`:material-pencil:{ .xl }`

**Reply by 91it** [reply]
Got it , so cool . thanks
